pkg/k8shandler/collection.go

```diff
@@ -5,36 +5,42 @@ import (
 	"io/ioutil"
 	"reflect"
 
+	"strings"
+	"time"
+
 	"github.com/openshift/cluster-logging-operator/pkg/utils"
 	"github.com/sirupsen/logrus"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/util/retry"
-	"strings"
-	"time"
 
 	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
 	sdk "github.com/operator-framework/operator-sdk/pkg/sdk"
 	apps "k8s.io/api/apps/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 )
 
+const (
+	clusterLoggingPriorityClassName = "cluster-logging"
+)
+
 var (
 	retryInterval = time.Second * 30
 	timeout       = time.Second * 1800
 )
 
-func CreateOrUpdateCollection(cluster *logging.ClusterLogging) (err error) {
+func (localLogging *ClusterLogging) CreateOrUpdateCollection() (err error) {
 
-	if err = createOrUpdateCollectionPriorityClass(cluster); err != nil {
+	cluster := localLogging.ClusterLogging
```

Comment:
if you update the function to be for `(cluster *logging.ClusterLogging)` does this line become unnecessary?

Reply:
This is really a 'stop gap' to make refactoring more digestible.  I would expect this line to be unnecessary after refactoring all the places where its needed